src/components/line-chart.js

```diff
@@ -47,18 +47,28 @@ const calcPrioritizedKeys = (dataPoints, countries) => {
     maxPerDataKey[`totalCases${country}`] = 0
     maxPerDataKey[`totalDeaths${country}`] = 0
     maxPerDataKey[`newDeaths${country}`] = 0
+    
+    maxPerDataKey[`newCases${country}PerCapita`] = 0
+    maxPerDataKey[`totalCases${country}PerCapita`] = 0
+    maxPerDataKey[`totalDeaths${country}PerCapita`] = 0
+    maxPerDataKey[`newDeaths${country}PerCapita`] = 0
+
     for (const entry of dataPoints) {
       if (entry[`newCases${country}`] > maxPerDataKey[`newCases${country}`]) {
         maxPerDataKey[`newCases${country}`] = entry[`newCases${country}`]
+        maxPerDataKey[`newCases${country}PerCapita`] = entry[`newCases${country}PerCapita`]
       }
       if (entry[`totalCases${country}`] > maxPerDataKey[`totalCases${country}`]) {
         maxPerDataKey[`totalCases${country}`] = entry[`totalCases${country}`]
+        maxPerDataKey[`totalCases${country}PerCapita`] = entry[`totalCases${country}PerCapita`]
       }
       if (entry[`totalDeaths${country}`] > maxPerDataKey[`totalDeaths${country}`]) {
         maxPerDataKey[`totalDeaths${country}`] = entry[`totalDeaths${country}`]
+        maxPerDataKey[`totalDeaths${country}PerCapita`] = entry[`totalDeaths${country}PerCapita`]
       }
       if (entry[`newDeaths${country}`] > maxPerDataKey[`newDeaths${country}`]) {
         maxPerDataKey[`newDeaths${country}`] = entry[`newDeaths${country}`]
+        maxPerDataKey[`newDeaths${country}PerCapita`] = entry[`newDeaths${country}$PerCapita`]
```

Comment:
Can't use the same if to check max. The population dividing could mean that it was not max in the per capita case.